03/app.js

```diff
@@ -7,12 +7,64 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
+function randomNumber(min, max) {
+    return Math.round((Math.random() * (max - min)) + min);
+}
 
+function getSum(a,b,c) {
+    a = Math.round(a);
+    b = Math.round(b);
+    c = Math.round(c);
+    let firstMax = 0;
+    let secondMax = 0;
+    if(a>=b || a>=c){
+        firstMax = a;
+        console.log('Przypisuje pierwszy max do ' + a);
+    }
+    if(b>=a || b>=c) {
+        if(firstMax == 0){
+            firstMax = b;
+            console.log('Przypisuje pierwszy max do ' + b);
+        } else {
+            secondMax = b;
+            console.log('Przypisuje drugi max do ' + b);
+        }
+    }
+    if(c>=a || c>=b){
+        secondMax = c;
+        console.log('Przypisuje drugi max do ' +c);
+    }
+    console.log(`${firstMax} ${secondMax} Suma dwóch największych liczb to: ${firstMax + secondMax}`);
+    // W pierwszym podpunkcie jest napisane, że ma być zwrócona wartość 2 największych liczb, a później jest mowa o użyciu funkcji w celu zwrócenia sumy z trzech, zakomentowałem zwrócenie dwóch które wyświetlam powyżej w konsoli, a funkcja będzie zwracać sume trzech liczb jak w linii 41
```

Comment:
Chodzi o to, aby z tych 3 liczb zrobić sumę, ale tylko dla 2 największych - o to chodzi w zadaniu.